src/main/java/Comments.java

```diff
@@ -25,9 +25,9 @@
   }
 
   public void foo() {
-    Type t = new Type("firstParameter", // this is supposed to be single line, using a lot of text to show off long lines
-        "secondParameter", // this is single line
-        "thirdParameter");
+    Type t = new Type("firstParameter", // this is supposed to be single line, using a lot of text to show off long line
+      "secondParameter", // this is single line
+      "thirdParameter");
```

Comment:
We use a double-indent for extended lines.

Reply:
This was one of the things that was hard to get right in terms of both the Eclipse and  IntelliJ formatters and enforcing with checkstyle, so I cheated by changing the example.  I did not see any examples of this style in the insight-brain repo, maybe it is more common on the nxrm side?

Reply:
> I did not see any examples of this style in the insight-brain repo, maybe it is more common on the nxrm side?

That is extremely surprising to me, as I come across them on a daily, perhaps even hourly basis when working with the insight-brain Java code.  In fact I just found one by opening the first that came to mind and looking at the first few dozen lines: https://github.com/sonatype/insight-brain/blob/master/insight-brain-service/src/main/java/com/sonatype/insight/brain/successmetrics/SuccessMetricsReportService.java#L43

Reply:
Hmm.  Well, my eyes glazed right over your example, so I guess I better figure out how to get the formatter/checker to do that properly until I get used to it.